tests/e2e/cucumber/features/app-provider/officeSuites.feature

```diff
@@ -19,10 +19,18 @@ Feature: Integrate with online office suites like Collabora and OnlyOffice
     And "Alice" creates a public link of following resource using the sidebar panel
       | resource         | role     | password |
       | OpenDocument.odt | Can edit | %public% |
-    And "Alice" logs out
     And "Anonymous" opens the public link "Link"
     And "Anonymous" unlocks the public link with password "%public%"
-    Then "Anonymous" should see the content "OpenDocument Content" in editor "Collabora"
+    Then "Anonymous" should be able to edit content of following resource
+      | resource         | type         | content                     |
+      | OpenDocument.odt | OpenDocument | Edited OpenDocument Content |
+    When "Alice" edits the public link named "Link" of resource "OpenDocument.odt" changing role to "Can view"
+    And "Anonymous" opens the public link "Link"
+    And "Anonymous" unlocks the public link with password "%public%"
```

Comment:
is this step required? cause i see this step has been already done above? please check